account/account.go

```diff
@@ -103,6 +103,11 @@ func GetAccount(ctx context.Context, cfg *config.Configuration, fetcher stored_r
 		return nil, errs
 	}
 
+	errs = account.Privacy.IPMasking.Validate(errs)
+	if len(errs) > 0 {
+		return nil, errs
```

Comment:
Is there a reasonable way to add a test that throws this error? Or is it challenging with this Validate function?

Reply:
Yes, added!